File: SetZeroes.cs
public class Solution {
    public void SetZeroes(int[][] matrix) {
        
        List<List<int>> cord = new List<List<int>>();
        
        for (int row = 0; row < matrix.Length; row++)
        {
            for (int col = 0; col < matrix[0].Length; col++)
            {
                if (matrix[row][col] == 0)
                {
                    cord.Add(new List<int>(2){row, col});
                }
            }
        }
        
        // Debug
        for (int i = 0; i < cord.Count; i++)
        {          
            for (int row = 0; row < matrix.Length; row++)
            {
                matrix[row][cord[i][1]] = 0;
            }
            
            for (int col = 0; col < matrix[0].Length; col++)
            {
                matrix[cord[i][0]][col] = 0;
            }
        }
    }
}

File: Python/setZeroes.py
class Solution:
    def setZeroes(self, matrix: List[List[int]]) -> None:
        """
        Do not return anything, modify matrix in-place instead.
        """
        w, h = len(matrix[0]), len(matrix)
        tempMatrix = [[0 for x in range(w)] for y in range(h)] 
        row = 0
        while row < len(matrix):
            col = 0
            while col < len(matrix[0]):
                tempMatrix[row][col] = matrix[row][col]
                col += 1
            row += 1

        row = 0
        while row < len(matrix):
            col = 0
            while col < len(matrix[0]):
                if matrix[row][col] == 0:
                    for y in range(h):
                        tempMatrix[y][col] = 0
                    for x in range(w):
                        tempMatrix[row][x] = 0
                col += 1
            row += 1

        row = 0
        while row < len(matrix):
            col = 0
            while col < len(matrix[0]):
                matrix[row][col] = tempMatrix[row][col]
                col += 1
            row += 1